core/sql/sqlmxevents/logmxevent_sq.cpp

```diff
@@ -49,7 +49,7 @@
 #endif
 #include "common/sql.error_event.pb.h"
 #include "common/sql.info_event.pb.h"
-#include "wrapper/amqpwrapper.h"
```

Comment:
This file could also be cleaned up/deleted. But lets do this separately after ensuring all the event logging has been ported to the current logmxevent_traf.cpp file completely.